common/buildings/07_government.txt

```diff
@@ -49,7 +49,7 @@ building_government_administration = {
 	
 	icon = "gfx/interface/icons/building_icons/building_government_administration.dds"
 	
-	required_construction = construction_cost_very_low
+	required_construction = construction_cost_low
```

Comment:
I actually quite like the very_low costs for admin buildings. It is a bit unrealistic for example for healthcare and stuff. But it is an expenditure anyways.